src/landpatterns/thermal-vias.stanza

```diff
@@ -0,0 +1,148 @@
+#use-added-syntax(jitx)
+defpackage jsl/landpatterns/thermal-vias:
+  import core
+  import jitx
+  import jitx/commands
+
+  import jsl/ensure
+  import jsl/errors
+  import jsl/design/introspection
+  import jsl/landpatterns/grid-planner
+
+doc: \<DOC>
+Thermal Via Interface
+
+This interface outlines the functions needed to create a thermal via
+grid. The idea is that the `make-thermal-vias` function gets called
+as a generator inside a `pcb-module` definition and targets one of
+the pads of a component instantiated in that module.
```

Comment:
Should we confirm that the component in question is also placed in code?

Reply:
Thats a good idea.